CONP_main_data_documentation.htm

```diff
@@ -1,40 +1,41 @@
+<br>
+<ul>
+<li>share phenotypic/genotypic data and methods in an unrestricted manner</li>
+<li>create large-scale databases impossible to achieve within a single institution</li>
+<li>facilitate the use of advanced multivariate analytic strategies</li>
+<li>train the next generation of computational neuroscience researchers and</li>
+<li>disseminate the results to the global community.</li>
+</ul>
+<br><br>
 <h2>2. Metadata</h2>
-
+<br>
 We use the DATS.json file format to store metadata about CONP datasets. DATS.json is a flexible machine-readable structure allowing for sophisticated representation of a wide range of different types of datasets, from neuroimaging to genomics.  A DATS.json file must be prepared for each dataset.
-
+<br>
 The CONP DATS.json schema includes the following required fields:
-
-- Dataset title and description
-- Creators of the dataset
-- DOI or other unique identifier
-- License under which the dataset is distributed
-- Keywords and specification of data type
-- Version of the dataset
-- Format of primary data files in dataset
-- Total number of files in the dataset
-- Size of dataset and unit used (usually GB)
-- Web address for original dataset
-
-
-
+<br>
+<ul>
+<li>Dataset title and description</li>
+<li>Creators of the dataset</li>
+<li>DOI or other unique identifier</li>
+<li>License under which the dataset is distributed</li>
+<li>Keywords and specification of data type</li>
+<li>Version of the dataset</li>
+<li>Format of primary data files in dataset</li>
+<li>Total number of files in the dataset</li>
+<li>Size of dataset and unit used (usually GB)</li>
+<li>Web address for original dataset</li>
+</ul>
+<br>
 The schema also includes many other recommended and optional fields.  The DATS dataset schema can be found <a href="https://github.com/datatagsuite/schema/blob/master/dataset_schema.json">here</a>, and details of CONP's expansions and descriptions of what to enter in each field <a href="./CONP_DATS_fields.md">here</a>.  The example DATS.json file for the 1000 Genomes Project <a href="https://github.com/conpdatasets/1000GenomesProject/blob/master/DATS.json">here</a> can be used as a template.  An interface allowing users to fill in required fields online is under development <a href="https://dats-creator.herokuapp.com/">here</a>.
-
+<br><br>
 <h2>3. Data files</h2>
-
+<br>
 CONP stores some datasets locally and provides access to others hosted at external resources.  We use the DataLad and GitAnnex software tools to manage data storage and access to external data.  This system allows users to construct links to datasets at locations of their choice, and for users to download only the files of particular interest to them from any given dataset.
-
+<br>
 A technical description of the installation and use of DataLad/GitAnnex to upload and download data can be found <a href="./datalad_dataset_addition_procedure.md">here</a>.
```

Comment:
This link seems to be broken.

Reply:
I have updated that to an absolute link; @PapillonMcGill had suggested relative links earlier and this was a relic of that.

Reply:
That link misses important pieces of doc, in particular regarding the Zenodo way to upload data. The information in https://github.com/CONP-PCNO/conp-dataset/blob/master/.github/CONTRIBUTING.md should be present in the documentation, and this CONTRIBUTING.md should be removed when merged.

Reply:
I have now updated the contents of that link, and made some further small changes to CONP_main_data_documentation.htm.  Let me know whether you are satisfied with these, and if so I will remove CONTRIBUTING.md.